Scripts/Embrace.cs

```diff
@@ -228,34 +315,60 @@ public void LogMessage(string message, EMBSeverity severity, Dictionary<string,
             provider.LogMessage(message, severity, properties, allowScreenshot);
         }
 
+        /// <summary>
+        /// Logs a breadcrumb.
+        /// Breadcrumbs track a user's journey through the application and will be shown on the timeline.
+        /// </summary>
+        /// <param name="message">the name of the breadcrumb to log</param>
         public void LogBreadcrumb(string message)
         {
             if (message == null) { NoNullsError(); return; }
             provider.LogBreadcrumb(message);
         }
 
+        /// <summary>
+        /// Ends the current session and starts a new one.
+        /// </summary>
+        /// <param name="clearUserInfo">if true, clears all the user info on the device</param>
         public void EndSession(bool clearUserInfo = false)
         {
             provider.EndSession(clearUserInfo);
         }
 
+        /// <summary>
+        /// Get the user identifier assigned to the device by Embrace.
+        /// </summary>
+        /// <returns>the device identifier created by Embrace</returns>
         public string GetDeviceId()
         {
             return provider.GetDeviceId();
         }
 
+        /// <summary>
+        /// Starts a view. There is a limit to 10 "started" views.
```

Comment:
@elanz-embrace Please verify this, need to add anything?

Reply:
we might use the word "open" instead of "started" for this, otherwise this feels correct.

Reply:
Changed to "open"